wheel/bdist_wheel.py

```diff
@@ -150,7 +151,11 @@ def get_tag(self):
         elif self.root_is_pure:
             plat_name = 'any'
         else:
-            plat_name = self.plat_name or get_platform()
+            # macosx contains system version in platform name so need special handle
+            if self.plat_name and not self.plat_name.startswith("macosx"):
+                plat_name = self.plat_name
+            else:
+                plat_name = get_platform(self.bdist_dir)
```

Comment:
Please add a blank line after every nested block (here and elsewhere).

Reply:
I have hope that I found all places.